sympy/assumptions/refine.py

```diff
@@ -32,7 +32,7 @@ def refine(expr, assumptions=True):
     x
 
     >>> refine(Q.real(x), Q.positive(x))
-    True
+    Q.real(x)
```

Comment:
`refine(Q.real(x), Q.positive(x))` returns `True` in current master branch.

Reply:
![refine_error](https://user-images.githubusercontent.com/61404154/107372718-bb880d00-6b0b-11eb-9213-cb53731c85a3.PNG)
 

@JSS95  But that's not correct .. If you run the code , it gives Q.real(x) not True..

Reply:
@ayushbisht2001 perhaps you are running `sympy-1.7.1` or an older version. I can confirm that on current master (1.8-dev), `True` is the result being returned. Further, if there was any discrepancy then I believe the doctests would be failing on current master.